src/app/config/config.dev.json

```diff
@@ -1,5 +1,7 @@
 {
   "api-url": "http://localhost:8080",
   "api-version": "v2",
-  "app-url": "http://localhost:4200"
+  "app-url": "http://localhost:4200",
+  "tree-node-counts-update": true,
```

Comment:
flags such as tree-node-counts-update have been moved to query.service, as we discussed earlier

Reply:
Yes, but then you cannot set them from outside, e.g, using Puppet. Default values can be in query.service, but you should be able to read the values from the json if present.

Reply:
The way I implemented it now allows you to omit these fields in the json, so puppet only needs to set them when they need to be overridden.

Reply:
If they can be omitted, then they should be omitted here, right?

Reply:
I mean we can still try to read the flags in query.service, since puppet might override config.prod.json some day, but since this is glowingbear code, we keep the config jsons unchanged

Reply:
I thought they could serve as examples, with the default values filled in. But I could also document the possible values in the readme instead.